eval/getexamples.py

```diff
@@ -0,0 +1,214 @@
+    "missing-word",  # orð vantar   omission    í Donalda > í þorpinu Donalda
+    "missing-words",  # fleiri en eitt orð vantar   omission    því betri laun > því betri laun hlýtur maður
+    "nonit4it",  # óskáletrað fyrir skáletrað       orðið qibt > orðið qibt
+    "noun4dem",  # nafnorð í stað ábendingarfornafns    grammar stærsta klukkan > sú stærsta
+    "noun4pro",  # nafnorð í stað fornafns  grammar menntun má nálgast > hana má nálgast
+    "past4pres",  # sögn í þátíð í stað nútíðar grammar þegar hún leigði spólur > þegar hún leigir spólur
+    "period4comma",  # punktur fyrir kommu  punctuation meira en áður. Hella meira í sig > meira en áður, hella meira í sig
+    "period4conjunction",  # punktur fyrir samtengingu  punctuation ...maður vill gera. Vissulega > ...maður vill gera en vissulega
+    "period4ex",  # punktur fyrir upphrópun punctuation Viti menn. > Viti menn!
+    "pers4dem",  # persónufornafn í staðinn fyrir ábendingarf.  grammar það > þetta
+    "pres4past",  # sögn í nútíð í stað þátíðar grammar Þeir fara út > Þeir fóru út
+    "pro4noun",  # fornafn í stað nafnorðs  grammar þau voru spurð > parið var spurt
+    "pro4reflexive",  # nafnorð í stað afturbeygðs fornafns grammar gefur orku til fólks í kringum það > gefur orku til fólks í kringum sig
+    "pro4reflexive",  # persónufornafn í stað afturbeygðs fn.   grammar Fólk heldur að það geri það hamingjusamt > Fólk heldur að það geri sig hamingjusamt
+    "punctuation",  # greinarmerki  punctuation hún mætti og hann var ekki tilbúinn > hún mætti en hann var ekki tilbúinn
+    "qm4ex",  # spurningarmerki fyrir upphrópun punctuation Algjört hrak sjálf? > Algjört hrak sjálf!
+    "reflexive4noun",  # afturbeygt fornafn í stað nafnorðs grammar félagið hélt aðalfund þess > félagið hélt aðalfund sinn
+    "reflexive4pro",  # afturbeygt fornafn í stað persónufornafns   grammar gegnum líkama sinn > gegnum líkama hans
+    "simple4cont",  # nútíð í stað vera að + nafnh. grammar ók > var að aka
+    "square4bracket",  # hornklofi fyrir sviga  punctuation [börnin] > (börnin)
+    "style",  # stíll   style   urðu ekkert frægir > urðu ekki frægir
+    "sub4ind",  # viðtengingarh. fyrir fh.  grammar Stjórnvöld vildu auka rétt borgara og geri þeim kleift > Stjórnvöld vildu auka rétt borgara og gera þeim kleift
+    "unicelandic",  # óíslenskuleg málnotkun    style   ...fer eftir persónunni... > ...fer eftir manneskjunni...
+    "upper4lower-proper",  # stór stafur í sérnafni þar sem hann á ekki að vera capitalization  Mál og Menning > Mál og menning
+    "wording",  # orðalag   wording ...gerðum allt í raun... > ...gerðum í raun allt...
+    "xxx",  # unclassified  unclassified
+    "zzz",  # to revisit    unannotated
+}
+
+
+def element_text(element: ET.Element) -> str:
+    """ Return the text of the given element,
+        including all its subelements, if any """
+    return "".join(element.itertext())
+
+def correct_spaces(tokens: List[Tuple[str, str]]) -> str:
```

Comment:
A more permissive signature would be `Iterable[Tuple[str, str]]`

Reply:
Done